dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1794,13 +1798,13 @@ def test_append_cat_fp(tmpdir, engine):
         pytest.param(
             pd.DataFrame({"x": pd.Categorical(["a", "b", "a"])}),
             marks=pytest.mark.xfail(
-                reason="https://issues.apache.org/jira/browse/ARROW-3652"
+                reason="https://issues.apache.org/jira/browse/ARROW-3652", strict=False
```

Comment:
Is this working for pyarrow 0.15+? If so, can you change these to be

```python
PYARROW_015 = LooseVersion(pa.__version__ >= "0.15.0"
strict=not PYARROW_0150
```

Just to ensure we don't regress on things.

Reply:
Good call - Thanks!